slack_sdk/proxy_env_variable_loader.py

```diff
@@ -13,8 +13,11 @@ def load_http_proxy_from_env(logger: logging.Logger = _default_logger) -> Option
         or os.environ.get("HTTP_PROXY")
         or os.environ.get("http_proxy")
     )
-    if proxy_url is not None:
-        logger.debug(
-            f"HTTP proxy URL has been loaded from an env variable: {proxy_url}"
-        )
+    if proxy_url is None or len(proxy_url.strip()) == 0:
+        # If the value is an empty string, the intention should be unsetting it
+        if len(proxy_url.strip()) == 0:
```

Comment:
This line does not work if the `proxy_url` is None. Can you verify it with a new test pattern in `tests/test_proxy_env_variable_loader.py`?

```python
>>> proxy_url = None
>>> len(proxy_url.strip())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'strip'
```

How about doing this way?

```python
    if proxy_url is None:
        return None
    if len(proxy_url.strip()) == 0:
        # If the value is an empty string, the intention should be unsetting it
        logger.debug("...")
        return None
```